### PR TITLE
[PowerMock] LPS 151992 

### PR DESCRIPTION
**Pull request by JanisZhang**
Hi @tinatian @dantewang,

This pr is to remove powermock in `modules/dxp....vldap-server-test`. Thanks.

https://issues.liferay.com/browse/LPS-151992

### CODE REVIEW

File: modules/dxp/apps/vldap/vldap-server-test/src/test/java/com/liferay/vldap/server/internal/portal/security/auth/SambaAuthTest.java [hunk 1]
```diff
@@ -28,19 +29,22 @@
...
 
 import org.mockito.Mockito;
 
-import org.powermock.modules.junit4.PowerMockRunner;
-
 /**
  * @author Jonathan McCann
  */
-@RunWith(PowerMockRunner.class)
 public class SambaAuthTest extends BaseVLDAPTestCase {
```
Comment (dantewang) [inline modules/dxp/apps/vldap/vldap-server-test/src/test/java/com/liferay/vldap/server/internal/portal/security/auth/SambaAuthTest.java:41]: @JanisZhang These tests are extending `BaseVLDAPTestCase` which still uses PowerMock. Could you also change that class and include it in this PR?

**Review by dantewang — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/vldap/vldap-server-test/src/test/java/com/liferay/vldap/server/internal/portal/security/auth/SambaAuthTest.java (hunk 1 above)
Comment (JanisZhang) [reply modules/dxp/apps/vldap/vldap-server-test/src/test/java/com/liferay/vldap/server/internal/portal/security/auth/SambaAuthTest.java:41]: @dantewang  Hi Dante, The reason I didn't include `BaseVLDAPTestCase` in this PR is that it still have one class `DirectoryTreeTest` (powermock **LdapUtil**)  extends BaseVLDAPTestCase. I still trying to find a way to replace LdapUtil in `DirectoryTreeTest` yet...

Comment (dantewang) [reply modules/dxp/apps/vldap/vldap-server-test/src/test/java/com/liferay/vldap/server/internal/portal/security/auth/SambaAuthTest.java:41]: @JanisZhang OK, let's wait for this to be done. If you can't find a way please let me know so we can look into it together. I'm closing this one for now.

**Comment by dantewang** [comment]
ci:close

**Pull request closed without merging**